ArticleTemplates/calloutTemplate.html

```diff
@@ -0,0 +1,11 @@
+<details class="article_callout">
+  <summary class="article_callout_heading">
+    <div class="article_callout_title">
+      __CALLOUT_TITLE__
+    </div>
+    __CALLOUT_DESCRIPTION__
+  </summary>
+  <form method="post" action="x-gu://participate/__CALLOUT_ID__" class="article_callout_body">
+    __CALLOUT_BODY
```

Comment:
Are you missing the trailing `__` on `__CALLOUT_BODY`?

Reply:
Oh yeah I am, thanks!

Reply:
... I'm wondering whether the form templates should be called `formXYZ` instead, their use in callouts is only circumstantial. What do you think?

Reply:
Good idea, they look useful, giving them more generic names leaves the possibility for reusing them elsewhere without having to worry about renaming them (and all the native work this'd entail) or using "callout" templates in a non-callout context.